schema.graphql

```diff
@@ -70,6 +75,7 @@ type Dataset {
   status: JobStatus
   inputPath: String
   uploadDateTime: String
+  fdrCounts: FdrCountsNumber
```

Comment:
I guess you should decide whether counts for all FDR levels should be returned by this field (then the use of this custom type `FdrCountsNumber` is perfectly fine), or a count for specific FDR level is returned (then the `level` property doesn't make much sense).

Based on the current usage of this field (we don't need counts for all FDR levels), I'd suggest putting it as a function here with `fdrLevel` parameter, s.t. it'd return just one number.